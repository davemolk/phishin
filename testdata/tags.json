{"success":true,"total_entries":19,"total_pages":1,"page":1,"data":[{"id":5,"name":"Costume","slug":"costume","group":"Set Content","color":"#888888","priority":5,"description":"Musical costume sequence","updated_at":"2019-02-10T06:58:41Z"},{"id":21,"name":"Audience","slug":"audience","group":"Song Content","color":"#888888","priority":19,"description":"Contribution from audience during performance","updated_at":"2019-02-10T06:58:42Z"}]}