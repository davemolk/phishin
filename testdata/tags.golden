Name:     Description:                                   Group:
Costume   Musical costume sequence                       Set Content
Audience  Contribution from audience during performance  Song Content
